docs/extend/extend-zowe-overview.md

```diff
@@ -1,56 +1,52 @@
 
-You can extend Zowe in the following ways:
+- [Extend Zowe CLI](#extend-zowe-cli)
+- [Extend Zowe API Mediation Layer](#extend-zowe-api-mediation-layer)
+  - [Dynamic API registration](#dynamic-api-registration)
+  - [Static API registration](#static-api-registration)
+- [Add a plug-in to the Zowe Desktop](#add-a-plug-in-to-the-zowe-desktop)
+- [Lifecycle extensions as Zowe address spaces](#lifecycle-extensions-as-zowe-address-spaces)
 
-- Extending the Zowe Command Line Interface.
-- Adding a REST API service to the API Mediation Layer. 
-- Adding a plug-in to the Zowe Desktop.
+**Note:** For more information on the architecture of Zowe, see [Zowe Architecture](../getting-started/zowe-architecture.md).
 
-## Extending the Zowe Command Line Interface
+## Extend Zowe CLI
 
-Command Line Interface extensions are able to provide new commands through their own plug-in, see [Developing a new plug-in](extend-cli/cli-developing-a-plugin.md).  There is a sample extension plug-in that is provided together with a tutorial, see [Installing the sample plug-in](extend-cli/cli-installing-sample-plugin.md).  
+Zowe CLI extenders can build plug-ins that provide new commands. Zowe CLI is built using Node.js and is typically run on a machine other than z/OS, such as a PC, where the CLI can be driven through a terminal or command prompt, or on an automation machine such as a DevOps pipeline orchestrator.
 
-The command line interface is built using Node.js and is typically run on a machine other than z/OS, such as a PC where it can be driven through a Terminal or command prompt, or on an automation machine such as a DevOps pipeline orchestrator. The API Mediation Layer and Zowe Desktop run on z/OS.  Support for running the API Mediation Layer and Zowe Desktop off platform might come in a future release of Zowe. To understand the architecture of Zowe, see [Zowe Architecture](../getting-started/zowe-architecture.md). 
+For more information about extending the Zowe CLI, see [Developing a new plug-in](extend-cli/cli-developing-a-plugin.md). This article includes a sample plug-in that is provided with the tutorial; see [Installing the sample plug-in](extend-cli/cli-installing-sample-plugin.md).
 
-## Adding a REST API service to the API Mediation Layer
+## Extend Zowe API Mediation Layer 
 
-The API Mediation Layer includes an API gateway that acts as a reverse proxy server through which API requests can be routed from clients on its northbound edge to z/OS servers on its southbound edge. The API gateway is extensible so you can add REST APIs for z/OS servers to its list of services. For information about how to onboard REST APIs, see [Onboarding Overview](extend-apiml/onboard-overview.md).  
+Zowe API Mediation Layer extenders can build and onboard additional API services to the API ML. REST APIs can register with the API Mediation Layer, which makes them available in the API Catalog and for routing through the API Gateway.  
```

Comment:
I'd propose to add the words "microservices ecosystem" at the end of the first sentence of the paragraph, as it is not a direct extension of the API ML (Discovery service, Gateway, Catalog, Zaas) but adding microservice to the discovery and catalog.

Reply:
Added your proposed language